### ISSUE TITLE
NSM setup doesn't wait for NSM components to become ready

### ISSUE THREAD

**Issue by d-uzlov** [networkservicemesh/deployments-k8s#9234]
## Expected Behavior

If NSM setup commands finish successfully, all NSM components are up and running.

## Current Behavior

We only make sure that NSM webhook is ready.

We have several NSM setup examples, and none of them check deployment status:
- https://github.com/networkservicemesh/deployments-k8s/tree/565602eed9af860c5a9b52c756b1a70603101cba/examples/basic
- https://github.com/networkservicemesh/deployments-k8s/tree/565602eed9af860c5a9b52c756b1a70603101cba/examples/multicluster
- https://github.com/networkservicemesh/deployments-k8s/tree/565602eed9af860c5a9b52c756b1a70603101cba/examples/interdomain/nsm
- https://github.com/networkservicemesh/deployments-k8s/tree/565602eed9af860c5a9b52c756b1a70603101cba/examples/sriov

## Steps to Reproduce

1. Mess up nsmgr config (for example, use non-existent image for container)
2. Do basic setup — finishes fine, because there is no check for nsmgr status
3. Run any test
4. Test fails

**Comment by denis-tingaikin** [comment]
its expected. Everything should work even if some of components started late. We're waiting webhook because it's needed for external clients.

So please report tests that dont work.

**Comment by d-uzlov** [comment]
I faced an issue here when running automated integration tests locally.
All of the tests failed for me. When investigating this, I found out that NSM setup failed because of issues with my local environment.
If we waited for the apps ready state in examples, such issues would be obvious, without digging up logs and describes.

I guess when running tests manually it wouldn't be as big of an issue, because I would probably figure out to check the NSM state.

If we specifically don't want to add waiting commands in the NSM setup, and expect that automated integration tests should always be run in a proper environment, then I guess we can just close this issue.
